aten/src/ATen/core/jit_type.h

```diff
@@ -1393,25 +1393,31 @@ inline TypePtr CompleteTensorType::fromNumberType(TypePtr typ) {
   } else if (typ->isSubtypeOf(BoolType::get())) {
     return CompleteTensorType::create(at::kLong, at::kCPU, {});
   }
-  AT_ERROR("unknown number type", typ->str());
+  AT_ERROR("unknown number type: ", typ->str());
```

Comment:
AT_ERROR is deprecated, please use TORCH_WARN

Reply:
sure, I think you mean TORCH_CHECK for the equivalent error though not TORCH_WARN?

Reply:
Sorry, ofc TORCH_CHECK